pkgs/development/libraries/jasper/default.nix

```diff
@@ -40,6 +40,7 @@ stdenv.mkDerivation rec {
     homepage = https://www.ece.uvic.ca/~frodo/jasper/;
     description = "JPEG2000 Library";
     platforms = platforms.unix;
+    license = licenses.bsd3;
```

Comment:
https://github.com/mdadams/jasper/blob/master/LICENSE seems closer to BSD-2
But it's actually known by SPDX as [JasPer-2.0](https://spdx.org/licenses/JasPer-2.0.html)

It's not defined in licenses.nix but and there is few chances this would be reused by another package so maybe just define it in-line in the package?